src/main/resources/application.yml

```
@@ -37,3 +37,6 @@ oauth:
     client-secret: ${KAKAO_CLIENT_SECRET}
     redirect-uri: ${KAKAO_REDIRECT_URI}
 
+google:
+  api:
+    key: ${GOOGLE_API_KEY}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)